### PR TITLE
SolGov SOO loadout, SolGov hardsuits.

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
Adds an SOO style job/loadout for the SolGov faction, called "Brigadier General".
This loadout is functionally identical to the standard SOO loadout, with a different outfit and title to distinguish them.

The PR also ensure SolGov Marines and Brigadier Generals are given command bold and ERT colour on the radio.
2 different bulletproof hardsuits has been added for the "gamma" Solgov Specops, with an overcoat for the Brigadier General.
Sprites tweaked from the very talented SomeAngryMiner over at Baystation12/Baystation12#30821

## Why It's Good For The Game
A lack of "SOO" style loadouts/jobs for SolGov was pointed out to me by an admin, who mentioned that such a character makes more sense to use for SolGov missions or events. Why would an NT officer be commanding the operation?

### Brigadier General
2021-06-15 00_20_59-Window
### New Solgov Specops armours
image

## Changelog
:cl:
add: SolGov 'SOO' loadout added for admins.
add: Bulletproof hardsuits, used by Specops Solgov.
tweak: Some SolGov jobs are now correctly displayed as special roles on the radio.
/:cl:

### CODE REVIEW

**Comment by Shadeykins** [comment]
Suggestion as TSF units may be called due to other reasons than a distress call. This also brings the naming schema in-line with everything else (the schema for ships/stations are to shorthand them to acronyms).

"Attention NSS Cyberiad, this is the TSF-NV Aurelia. Boarding shuttles have been scrambled and marines are en-route to your location. Standby for boarding."

File: code/modules/response_team/ert.dm [hunk 1]
```diff
@@ -301,6 +301,22 @@ GLOBAL_VAR_INIT(ert_request_answered, FALSE)
 /datum/response_team/gamma/announce_team()
 	GLOB.event_announcement.Announce("Attention, [station_name()]. We are sending a code GAMMA elite Emergency Response Team. Standby.", "ERT En-Route")
 
+/datum/response_team/solgov
+	security_outfit = /datum/outfit/job/centcom/response_team/solgov
+	command_outfit = /datum/outfit/job/centcom/response_team/solgov/lieutenant
+	borg_path = /mob/living/silicon/robot/ert/red
+
+/datum/response_team/solgov/announce_team()
+	GLOB.event_announcement.Announce("Attention, [station_name()], this is the TSF navy vessel Aurelia. We have detected your distress call, a squad of Marines is inbound. Standby.", "Marines En-Route")
```
Comment (S34NW) [inline code/modules/response_team/ert.dm:310]:
```suggestion
	GLOB.event_announcement.Announce("Attention, [station_name()], this is the TSF-NV Aurelia. Boarding shuttles have been scrambled and marines are en-route to your location. Standby for boarding.", "Marines En-Route")
```

File: code/modules/response_team/ert.dm [hunk 2]
```diff
@@ -301,6 +301,22 @@ GLOBAL_VAR_INIT(ert_request_answered, FALSE)
...
+
+/datum/response_team/solgovspecops
+	security_outfit = /datum/outfit/job/centcom/response_team/solgov/elite
+	command_outfit = /datum/outfit/job/centcom/response_team/solgov/elite/lieutenant
+	borg_path = /mob/living/silicon/robot/ert/gamma
+
+/datum/response_team/solgovspecops/announce_team()
+	GLOB.event_announcement.Announce("Attention, [station_name()], this is the TSF navy vessel Aurelia. We have detected your distress call, a squad of elite SpecOps Marines is inbound. Standby.", "Marines En-Route")
```
Comment (S34NW) [inline code/modules/response_team/ert.dm:318]:
```suggestion
	GLOB.event_announcement.Announce("Attention, [station_name()], this is the TSF-NV Aurelia. Boarding shuttles have been scrambled and marines are en-route to your location. Standby for boarding.", "Marines En-Route")
```

**Review by S34NW — COMMENTED** [review]
Announcement changes

File: code/game/machinery/tcomms/nttc.dm [hunk 3]
```diff
@@ -119,7 +124,9 @@
 	/// List of ERT jobs
 	var/list/ert_jobs = list("Emergency Response Team Officer", "Emergency Response Team Engineer", "Emergency Response Team Medic", "Emergency Response Team Leader", "Emergency Response Team Member")
 	/// List of CentComm jobs
-	var/list/cc_jobs = list("Nanotrasen Navy Officer", "Special Operations Officer", "Syndicate Officer")
+	var/list/cc_jobs = list("Nanotrasen Navy Officer", "Special Operations Officer", "Syndicate Officer", "Solar Federation Brigadier General")
+	/// List of SolGov Marine jobs
+	var/list/tsf_jobs = list( "Solar Federation Specops Lieutenant", "Solar Federation Specops Marine", "Solar Federation  Lieutenant", "Solar Federation Marine")
```
Comment (dearmochi) [inline code/game/machinery/tcomms/nttc.dm:129]:
```suggestion
	var/list/tsf_jobs = list("Solar Federation Specops Lieutenant", "Solar Federation Specops Marine", "Solar Federation Lieutenant", "Solar Federation Marine")
```

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 4]
```diff
@@ -87,6 +87,11 @@
 	armor = list("melee" = 80, "bullet" = 80, "laser" = 50, "energy" = 50, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
 	flags =  STOPSPRESSUREDMAGE | THICKMATERIAL
 
+/obj/item/clothing/head/helmet/space/deathsquad/beret/solgov
+	name = "Trans-Solar Federation commander's beret"
```
Comment (dearmochi) [inline code/modules/clothing/spacesuits/miscellaneous.dm:91]:
```suggestion
	name = "\improper Trans-Solar Federation commander's beret"
```

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 5]
```diff
@@ -99,6 +104,12 @@
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 	w_class = WEIGHT_CLASS_NORMAL
 
+/obj/item/clothing/suit/space/deathsquad/officer/solgov
+	name = "Trans-Solar Federation commander's jacket"
```
Comment (dearmochi) [inline code/modules/clothing/spacesuits/miscellaneous.dm:108]:
```suggestion
	name = "\improper Trans-Solar Federation commander's jacket"
```

File: code/modules/response_team/ert_outfits.dm [hunk 6]
```diff
@@ -534,3 +534,130 @@
...
+		/obj/item/reagent_containers/hypospray/autoinjector/survival = 1
+	)
+	cybernetic_implants = list(
+		/obj/item/organ/internal/cyberimp/eyes/hud/security,
+		/obj/item/organ/internal/cyberimp/chest/nutriment,
+		/obj/item/organ/internal/cyberimp/brain/anti_stun/hardened,
+		/obj/item/organ/internal/cyberimp/arm/flash,
+		/obj/item/organ/internal/eyes/cybernetic/shield
+	)
+/datum/outfit/job/centcom/response_team/solgov/elite/lieutenant
```
Comment (dearmochi) [inline code/modules/response_team/ert_outfits.dm:641]:
```suggestion
	)

/datum/outfit/job/centcom/response_team/solgov/elite/lieutenant
```

**Review by dearmochi — APPROVED** [review]
Few nits. OORAH

File: code/modules/response_team/ert.dm [hunk 7]
```diff
@@ -301,6 +301,22 @@ GLOBAL_VAR_INIT(ert_request_answered, FALSE)
...
+/datum/response_team/solgov/announce_team()
+	GLOB.event_announcement.Announce("Attention, [station_name()], this is the TSF-NV Aurelia. Boarding shuttles have been scrambled and marines are en-route to your location. Standby for boarding.", "Marines En-Route")
+
+/datum/response_team/solgovspecops
+	security_outfit = /datum/outfit/job/centcom/response_team/solgov/elite
+	command_outfit = /datum/outfit/job/centcom/response_team/solgov/elite/lieutenant
+	borg_path = /mob/living/silicon/robot/ert/gamma
+
+/datum/response_team/solgovspecops/announce_team()
+	GLOB.event_announcement.Announce("Attention, [station_name()], this is the TSF-NV Aurelia. Boarding shuttles have been scrambled and marines are en-route to your location. Standby for boarding.", "Marines En-Route")
```
Comment (S34NW) [inline code/modules/response_team/ert.dm:318]:
```suggestion
	GLOB.event_announcement.Announce("Attention, [station_name()], this is the TSF-NV Aurelia. Boarding shuttles have been scrambled and specops marines are en-route to your location. Standby for boarding.", "Marines En-Route")
```

**Comment by kenken244** [comment]
Solgov is a multiracial organization, they should spawn in as random races. Or have the ability to choose. But just stinky hoomans

**Comment by AffectedArc07** [comment]
> Solgov is a multiracial organization, they should spawn in as random races. Or have the ability to choose. But just stinky hoomans

So is Nanotrasen given how their comms officers and SOOs are always different species. Doesn't mean their marines are (DS and ERT are always human). Why should we tweak this for just solgov?

The entire spawning system isn't being refactored because one man doesn't want only "stinky hoomans".

**Comment by Spacemanspark** [comment]
Very nice.

**Review by variableundefined — APPROVED** [review]

**Pull request merged**